### ISSUE TITLE
[Bug] Share icon in custom PWA toolbar has wrong color

### ISSUE THREAD

**Issue by topotropic** [mozilla-mobile/fenix#8966]
## Steps to reproduce
1. Install a PWA, for example paperplanes.world
2. View toolbar

### Expected behavior
Share icon has correct color

### Actual behavior
Share icon is white on light background

### Device information

* Android device: ?
* Fenix version: ?

<img width="471" alt="image" src="https://user-images.githubusercontent.com/353960/75868776-dedd0f00-5e08-11ea-9738-b057f078a476.png">

**Comment by LaurentiuApahideanSV** [comment]
I can confirm that the issue is reproducible using the latest version of Firefox Preview Nightly 3/5 #20650607.

Device:
Samsung Galaxy S8+ (Android 8.0.0) and OnePlus 6T (Android 9)

**Comment by AndiAJ** [comment]
**Reproducible** on the latest on the latest **`Nightly Build 201111`** using the following devices:
• Google Pixel 3a (Android 11)
• Huawei Mate 20 Lite (Android 10)
• OnePlus A3 (Android 6.0.1)

**Comment by eliserichards** [comment]
<img width="371" alt="image" src="https://user-images.githubusercontent.com/43795363/113460000-dfb1ed00-93dc-11eb-91f6-d491a661c844.png">

I'm not seeing the Share button (or the toolbar)

**Comment by AndiAJ** [comment]
Still reproducible on **`Firefox 88.1.2`** using a OnePlus A3 (Android 6.0.1)

**Comment by SoftVision-LorandJanos** [comment]
I checked the issue using the latest Nightly 101.0a1 (2022-04-29) build by opening the page in a custom tab.
As far as I know the toolbar is not displayed in PWA, as mentioned in @eliserichards's comment above.
The share icon is clearly visible with both light and dark themes:
![Screenshot_20220429-164712](https://user-images.githubusercontent.com/48829350/165958625-24fa3026-7999-4a77-b860-c56b5c4796e0.png)
![Screenshot_20220429-165106](https://user-images.githubusercontent.com/48829350/165958630-24ee3a66-c8c0-47f5-a296-d4f95c637599.png)
Closing the ticket as fixed.
